test/test_model.py

```diff
@@ -229,9 +381,13 @@ def cleanup():
         f.write('#!/bin/bash\n{ echo -n $(basename $0); for s in "$@"; do echo -n \\;$s; done; echo; } >> $(dirname $0)/calls.txt\n' + content)
     os.chmod(test_case.fake_script_path / name, 0o755)
 
-def fake_script_calls(test_case):
-    with open(test_case.fake_script_path / 'calls.txt') as f:
-        return [line.split(';') for line in f.read().splitlines()]
+def fake_script_calls(test_case, clear=False):
+    with open(test_case.fake_script_path / 'calls.txt', 'r+') as f:
+        calls = [line.split(';') for line in f.read().splitlines()]
+        if clear:
+            f.truncate(0)
```

Comment:
Nice idea.

Reply:
Thanks, this should be useful in other places as well.